--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,8 +104,8 @@
 
 # We copy the project binary to the 'bin' directory
 INSTALL (TARGETS ${PROJECT_NAME} DESTINATION bin)
-INSTALL (DIRECTORY "${PROJECT_SOURCE_DIR}/example/" DESTINATION example)
-INSTALL (FILES LICENSE README.md DESTINATION bin/..)
+# INSTALL (DIRECTORY "${PROJECT_SOURCE_DIR}/example/" DESTINATION example)
+# INSTALL (FILES LICENSE README.md DESTINATION bin/..)
 
 # cmake_system_name for mac is Darwin, i want to change that
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
